docs/source/docker/advanced/object_level.rst

```diff
@@ -0,0 +1,127 @@
+.. _ref-docker-object-level:
+
+Object Level
+============
+Lightly does not only work on full images but also on an object level. This 
+workflow is especially useful for datasets containing small objects or multiple
+objects in each image and provides the following benefits over the full image 
+workflow:
+
+- Analyze a dataset based on individual objects
+- Find a diverse set of objects in the dataset
+- Find images that contain objects of interest
+- Full control over type of objects to process
+- Ignore uninteresting background regions in images
+- Automatic cropping of objects from the original image
+
+
+Prerequisites
+-------------
+In order to use the object level workflow with Lightly, you will need the
+following things:
+
+- The installed Lightly docker (see :ref:`ref-docker-setup`)
+- A dataset with a configured datasource (see :ref:`ref-docker-with-datasource-datapool`)
+
+Lightly also needs to know which objects it should process. You can provide
+this information by uploading a set of object predictions to your datasource.
+Please follow the instructions here (TODO INSERT REFERENCE) on how to upload 
+predictions. Make sure that you upload object detection predictions and register
+them in the `tasks.json` file.
```

Comment:
Since the prediction stuff is rather complicated and error prone I'd suggest that we highlight here quickly your folder structure?